### PR TITLE
checklist/updates: few comment fixes for epoch handling

### PR DESCRIPTION
**Pull request by dustymabe**
- checklist/updates: fix `date` invocation
- checklist/updates: add some more clarity to epoch handling
- checklist/updates: update date command for last-modified field

### CODE REVIEW

File: ISSUE_TEMPLATE/testing.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 
 - [ ] Update updates metadata, editing `updates/testing.json` and replacing the oldest rollout with your new one:
   - [ ] Set `version` field to the new version
-  - [ ] Set `start_epoch` field to a future timestamp for the rollout start (e.g. `date -s <DATE> +%s`)
+  - [ ] Set `start_epoch` field to a future timestamp for the rollout start (e.g. `date -d <DATE> +%s`)
   - [ ] Set `start_percentage` field to `0.0`
   - [ ] Set `duration_minutes` field to a reasonable rollout window (e.g. `2880` for 48h)
   - [ ] Update the `last-modified` field to current time (e.g. `date -u -Is`)
```
Comment (jlebon) [inline ISSUE_TEMPLATE/testing.md]: Want to tweak this invocation too as in https://github.com/coreos/fedora-coreos-streams/pull/17#discussion_r322761892 ?

Comment (dustymabe) [reply ISSUE_TEMPLATE/testing.md]: 👍

**Comment by dustymabe** [comment]
pushed a new update

**Comment by dustymabe** [comment]
@lucab - considering the first commit in this PR (and assuming the date command on your system accepts the same options as the date command on mine) you might want to check the local time on your laptop :)

**Pull request merged**